src/translations/pt/coreComponents.js

```diff
@@ -0,0 +1,611 @@
+export default {
+  title: "Componentes principais",
+  description:
+    "Componentes são os principais blocos de construção para interfaces de usuário. A construção de uma biblioteca de componentes reutilizáveis aprimora o fluxo de trabalho de desenvolvimento do produto, reduzindo o design e a dívida tecnológica e acelerando o processo. Os componentes principais não podem ser desmembrados em peças granulares sem perder seu significado.",
+  sections: {
+    "cc-avatar": {
+      title: "Avatar",
+      description:
+        "Avatares são usados para mostrar uma miniatura de uma foto do usuário ou uma representação visual de qualquer outro tipo de conteúdo.",
+      checklist: {
+        "cc-avatar-image": {
+          title: "Imagem",
+          description:
+            "Os avatares devem mascarar uma imagem em sua forma e funcionar com qualquer tamanho de imagem, já que podem receber essa imagem de fontes de dados desconhecidas.",
```

Comment:
Would recommend "adequar" ou "comportar" instead of "marcarar". Also, "aceitar" instead of "funcionar"
"Os avatares devem adequar imagens à sua forma... e aceitar qualquer tamanho..."